### PR TITLE
Add shortcuts for screenshot, text log, and muting

### PR DESCRIPTION
**Pull request by x-ji**
screenshot: s
text log: l
muting: m

Additionally, append the current second to the screenshot name so that it's easy to make multiple screenshots within a minute.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -34,13 +34,13 @@
       </section>
       <section class="navbar-section">
         <div id="volume-control" class="mr-2 hidden-until-game-start">
-          <i id="volume-control-icon" class="fa fa-volume-up tooltip tooltip-bottom" data-tooltip="ミュート"></i>
+          <i id="volume-control-icon" class="fa fa-volume-up tooltip tooltip-bottom" data-tooltip="ミュート (m)"></i>
```
Comment (kichikuou) [inline docs/index.html:37]: Would you add these shortcut indicators in `en.html` too?

File: docs/index.html [hunk 2]
```diff
@@ -34,13 +34,13 @@
...
           <input id="volume-control-slider" type="range" class="tooltip tooltip-bottom" data-tooltip="音量">
         </div>
 
-        <button id="screenshot-button" class="btn btn-link tooltip tooltip-bottom mr-2 hidden-until-game-start" data-tooltip="スクリーンショット"><i class="fa fa-camera"></i></button>
+        <button id="screenshot-button" class="btn btn-link tooltip tooltip-bottom mr-2 hidden-until-game-start" data-tooltip="スクリーンショット (s)"><i class="fa fa-camera"></i></button>
```
Comment (kichikuou) [inline docs/index.html]: Currently I'm working on "Text Skip" feature and would like to save 's' shortcut for that :)

How about using 'p' for screenshot?

**Review by kichikuou — COMMENTED** [review]
Thank you for the PR!

File: docs/index.html (hunk 2 above)
Comment (x-ji) [reply docs/index.html]: Sure, changed :)

File: docs/index.html (hunk 1 above)
Comment (x-ji) [reply docs/index.html:37]: Sure. Added.

**Comment by kichikuou** [comment]
Thanks!

**Pull request merged**